### PR TITLE
[EEG Browser] Add ability to view annotations in event panel and overlaid on signal viewer

### PR DESCRIPTION
**Pull request by jesscall**
## Brief summary of changes
_**Disclaimer:**_ This PR is blocked by #7811 and will need to be rebased after its merge.

The changes in this PR fill in a few gaps in the code and allow for annotation data to be viewed in the EEG Browser. Annotations are displayed in orange in the `Events / Annotation Panel` and when selected are shown as an overlay in the signal viewer.

Screen Shot 2021-11-12 at 1 27 09 PM

#### Testing instructions (if applicable)

1. Reach out to @jesscall on slack to get access to annotation data / SQL patches
2. Insert the SQL into your DB and place the files on your VM.
3. Navigate to the `EEG Browser`
4. Make sure that you can see and interact with annotations data.

### CODE REVIEW

**Comment by christinerogers** [comment]
For clarification - does this commit Laetitia's atlas code to main, or is it your new fixes on top of that ?

**Comment by jesscall** [comment]
@christinerogers 

Laetitia's ATLAS code fetches the annotations data from the `annotation.tsv` file. 

Since @AlexandraLivadas 's PR #7433 was merged adding back-end structure for annotations, I implemented a solution that gets the info from the DB tables rather than reading from the tsv file. 

TLDR; The code in this PR is inspired by ATLAS code, but has a different implementation.

Hope that's clear?

**Comment by jesscall** [comment]
@christinerogers Also for clarification: adding new annotations via front-end is not part of this PR. There are still gaps in the code. PR for that coming soon.

**Comment by jesscall** [comment]
rebased, no longer blocked

**Comment by laemtl** [comment]
Replaced by #8235
